test/unit/operations/list_databases.test.ts

```diff
@@ -0,0 +1,42 @@
+import { expect } from 'chai';
+
+import { ListDatabasesOperation, MongoDBNamespace } from '../../mongodb';
+
+const mockDB = {
+  s: {
+    namespace: {
+      withCollection() {
+        return new MongoDBNamespace('test', 'test');
+      }
+    }
+  }
+};
+
+describe('ListDatabasesOperation', function () {
+  describe('#constructor', function () {
+    context('when nameOnly is provided', function () {
+      context('when nameOnly is true', function () {
+        const operation = new ListDatabasesOperation(mockDB, { nameOnly: true });
```

Comment:
nit: don't initialize variables outside of before hooks or tests.

Even when a test is skipped, any code outside the test blocks or hooks runs when the suite is loaded.  Consequently, if we ever need to run the debugger on a function that is called frequently outside of tests like this, we can't easily rely on just dropping breakpoints inside the relevant function, because we'll hit the same breakpoint for tests that are skipped.